src/plugins/CesiumUsdSchemas/src/CesiumUsdSchemas/wrapTileset.cpp

```diff
@@ -27,148 +27,148 @@ WRAP_CUSTOM;
+_CreateEnableFrustumCullingAttr(CesiumTileset &self,
                                       object defaultVal, bool writeSparsely) {
     return self.CreateEnableFrustumCullingAttr(
         UsdPythonToSdfType(defaultVal, SdfValueTypeNames->Bool), writeSparsely);
 }
         
 static UsdAttribute
-_CreateEnableFogCullingAttr(CesiumTilesetAPI &self,
+_CreateEnableFogCullingAttr(CesiumTileset &self,
                                       object defaultVal, bool writeSparsely) {
     return self.CreateEnableFogCullingAttr(
         UsdPythonToSdfType(defaultVal, SdfValueTypeNames->Bool), writeSparsely);
 }
         
 static UsdAttribute
-_CreateEnforceCulledScreenSpaceErrorAttr(CesiumTilesetAPI &self,
+_CreateEnforceCulledScreenSpaceErrorAttr(CesiumTileset &self,
                                       object defaultVal, bool writeSparsely) {
     return self.CreateEnforceCulledScreenSpaceErrorAttr(
         UsdPythonToSdfType(defaultVal, SdfValueTypeNames->Bool), writeSparsely);
 }
         
 static UsdAttribute
-_CreateCulledScreenSpaceErrorAttr(CesiumTilesetAPI &self,
+_CreateCulledScreenSpaceErrorAttr(CesiumTileset &self,
                                       object defaultVal, bool writeSparsely) {
     return self.CreateCulledScreenSpaceErrorAttr(
         UsdPythonToSdfType(defaultVal, SdfValueTypeNames->Float), writeSparsely);
 }
         
 static UsdAttribute
-_CreateSuspendUpdateAttr(CesiumTilesetAPI &self,
+_CreateSuspendUpdateAttr(CesiumTileset &self,
                                       object defaultVal, bool writeSparsely) {
     return self.CreateSuspendUpdateAttr(
         UsdPythonToSdfType(defaultVal, SdfValueTypeNames->Bool), writeSparsely);
 }
         
 static UsdAttribute
-_CreateSmoothNormalsAttr(CesiumTilesetAPI &self,
+_CreateSmoothNormalsAttr(CesiumTileset &self,
                                       object defaultVal, bool writeSparsely) {
     return self.CreateSmoothNormalsAttr(
         UsdPythonToSdfType(defaultVal, SdfValueTypeNames->Bool), writeSparsely);
 }
         
 static UsdAttribute
-_CreateShowCreditsOnScreenAttr(CesiumTilesetAPI &self,
+_CreateShowCreditsOnScreenAttr(CesiumTileset &self,
                                       object defaultVal, bool writeSparsely) {
     return self.CreateShowCreditsOnScreenAttr(
         UsdPythonToSdfType(defaultVal, SdfValueTypeNames->Bool), writeSparsely);
 }
 
 static std::string
-_Repr(const CesiumTilesetAPI &self)
+_Repr(const CesiumTileset &self)
 {
     std::string primRepr = TfPyRepr(self.GetPrim());
     return TfStringPrintf(
-        "Cesium.TilesetAPI(%s)",
+        "Cesium.Tileset(%s)",
         primRepr.c_str());
 }
 
 } // anonymous namespace
 
-void wrapCesiumTilesetAPI()
+void wrapCesiumTileset()
 {
-    typedef CesiumTilesetAPI This;
+    typedef CesiumTileset This;
 
-    class_<This, bases<UsdAPISchemaBase> >
-        cls("TilesetAPI");
+    class_<This, bases<UsdTyped> >
+        cls("Tileset");
```

Comment:
You may need to rerun `regenerate_schema.bat`. It looks like `tokens.h` and `wrapTileset.cpp` have a few more changes after I run it.

Reply:
Good call. Fixed.